### ISSUE TITLE
Windows version require SH

### ISSUE THREAD

**Issue by qw569** [sqshq/sampler#45]
Windows version required sh but default command interpreter in Windows OS is cmd.exe

SAMPLING FAILURE exec: "sh": executable file not found in %PATH% 

Исполняемый файл под Windows, пытается запустить внешние программы через SH. Командным интерпретатором в операционных системах Windows является cmd.exe.

**Comment by sqshq** [comment]
Yes, right now the windows support is experimental and relies on the sh interpreter. That's why it is [recommended](https://github.com/sqshq/sampler#windows-experimental) to use non-default emulators, e.g. Cmder

I don't have a Windows machine myself, so let's keep the issue for future in case someone will be interested to implement

**Comment by Welding-Torch** [comment]
Problem still persists in 2024.
